metrics/metrics.go

```diff
@@ -12,11 +12,11 @@ type ClientMetrics struct {
 }
 
 type CatalystAPIMetrics struct {
-	UploadVODRequestCount    prometheus.Counter
-	UploadVODSuccessCount    prometheus.Counter
-	UploadVODFailureCount    *prometheus.CounterVec
-	UploadVODPipelineResults *prometheus.CounterVec
-	TranscodeSegmentDuration prometheus.Histogram
+	UploadVODRequestCount        prometheus.Counter
+	UploadVODSuccessCount        prometheus.Counter
```

Comment:
Could we combine SuccessCount and FailureCount into CompletionCount and then have a "success" bool label on it?

Reply:
Done! https://github.com/livepeer/catalyst-api/pull/265/commits/dcef659a0caed0cf339ddc1e77f1d1ed11296501?diff=split&w=1

I also made it a summary instead, which contains a count but will also measure the request latency! Also refactored the handler a bit so we don't have to repeat measurement code so much (there was one that we missed)